hpms/src/test/java/gov/cms/ab2d/hpms/service/HPMSManualModeTest.java

```diff
@@ -0,0 +1,100 @@
+package gov.cms.ab2d.hpms.service;
+
+import gov.cms.ab2d.common.model.Contract;
+import gov.cms.ab2d.common.repository.ContractRepository;
+import gov.cms.ab2d.common.util.AB2DPostgresqlContainer;
+import gov.cms.ab2d.hpms.SpringBootTestApp;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.time.OffsetDateTime;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest(classes = SpringBootTestApp.class)
+@TestPropertySource(locations = "/application.hpms.properties")
+@Testcontainers
+class HPMSManualModeTest {
+
+    public static final String TEST_CONTRACT_NUMBER = "X1234";
+    @Autowired
+    private ContractRepository contractRepository;
```

Comment:
Use ContractService instead

Reply:
Not for test code.  The ContractService does not provide a way of deleting contracts.